### ISSUE TITLE
Add recommended role_name

### ISSUE THREAD

**Issue by staticdev** [gantsign/ansible-role-visual-studio-code#242]
Hello gantsign, after latest ansible-lint, now all playbooks that use [ansible-role-visual-studio-code](https://github.com/gantsign/ansible-role-visual-studio-code) will fail until the name on Galaxy is `role_name: visual_studio_code` on meta/main.yml:
https://github.com/ansible/ansible-lint/issues/3566#event-9539563617

Would you mind changing the name?

**Comment by freemanjp** [comment]
Yeah, I admit this is a pain, a few years ago Ansible changed the convention for role names from kebab case to snake case (without providing any way to migrate to the new naming convention). I can't change the name in Galaxy now because it'll break anyone relying on the current name.

Ansible Galaxy is pushing everyone to move their roles into collections. When I get time to do that, I'll update to the new naming convention, as that'll be a breaking change anyway.

In the meantime, I suggest you update your Ansible Lint config to suppress role name errors as I do here: https://github.com/gantsign/ansible-role-visual-studio-code/blob/master/.ansible-lint#L3.
